### DISCUSSION TITLE
Maximum Number of Files

### DISCUSSION THREAD

**Discussion by nognkantoor** [zim-desktop-wiki/zim-desktop-wiki#2597] [Q&A] [answered]
All:

* Is there a maximum number of files that can be in a ZIM Wiki?
* Is there a maximum size for an individual file?
* Is there a maximum size for all files within the zim-wiki?

Asking because ZIM-wiki is fairly slow, and I'm not sure if it is due to my platform, or the size of my wiki.
Current size is 50,000 .txt files, 100 MB, and 850 MB, respectively.
Projected size is ±1,000,000 .txt files, 100 MB, and ±5 GB, respectively.

**Comment by jaap-karssenberg** [comment, marked as answer]
Please see
https://github.com/zim-desktop-wiki/zim-desktop-wiki/wiki/Performance-large-notebook

**Reply by nognkantoor** [reply]
I didn't think of "Performance" as a synonym. 

I am looking at _far more files than the anticipated use case_.
I'll have to rethink how to do my project.
